politeiad/backendv2/tstorebe/plugins/pi/cmds.go

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package pi
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sort"
+	"strconv"
+	"time"
+
+	backend "github.com/decred/politeia/politeiad/backendv2"
+	"github.com/decred/politeia/politeiad/backendv2/tstorebe/store"
+	"github.com/decred/politeia/politeiad/plugins/pi"
+	"github.com/decred/politeia/politeiad/plugins/ticketvote"
+	"github.com/decred/politeia/util"
+)
+
+const (
+	pluginID = pi.PluginID
+
+	// Blob entry data descriptors
+	dataDescriptorBillingStatus = pluginID + "-billingstatus-v1"
+)
+
+// cmdBillingStatus sets record's billing status.
```

Comment:
```suggestion
// cmdBillingStatus sets a proposal's billing status.
```

When we're inside of the pi plugin we can refer to proposals by name instead of the generic "record" term.  CMS has different types of records (invoices, DCCs) and pi might in the future as well.